### PR TITLE
Readme revamp and Route53 tutorial

### PR DESCRIPTION
**Pull request by ytsarev**
* Split README to readable sections
* Create CONTRIBUTING.md
* Create CODEOWNERS
* Add Route53 tutorial together with reference terraform code, sample k8gb values yaml files and Gslb sample CR

### CODE REVIEW

**Comment by ytsarev** [comment]
guys, it is WIP, would love to get early feedback on the overall direction with docs

@somaritane it's inline on what we discussed in person
@donovanmuller I reused the abstract from your brilliant kubecon talk proposal
@k0da @kuritka dudes, any feedback/ideas is highly appreciated

File: docs/local.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+Clean up your local development clusters with
+```shell script
+make destroy-full-local-setup
+```
+
+## Sample demo
+
+### Round Robin
+
+Both clusters have [podinfo](https://github.com/stefanprodan/podinfo) installed on the top where each
```
Comment (kuritka) [inline docs/local.md:112]:
```suggestion
Both clusters have [podinfo](https://github.com/stefanprodan/podinfo) installed on the top, where each
```

**Review by kuritka — CHANGES_REQUESTED** [review]
nice...! 

_*I found some minimal typos_

File: README.md [hunk 2]
```diff
@@ -12,605 +12,61 @@
...
+k8gb focuses on load balancing traffic across geographically dispersed Kubernetes clusters using multiple load balancing strategies to meet requirements such as region failover for high availability.
 
-Hope you enjoyed the ride!
+Global load balancing for any Kubernetes Service can now be enabled and managed by any operations or development teams in the same Kubernetes native way as any other custom resource.
 
-If anything unclear or is going wrong feel free to contact us at https://github.com/AbsaOSS/k8gb/issues. We will appreciate any feedback/bug report and Pull Requests are welcome.
+## Key Differentiators
 
-For more advanced technical documentation and fully automated local installation steps, see below.
+* Load balancing is based on timeproof DNS protocol which is perfect for global scope and extremely reliable
```
[the remark this answers is not in the document]
Comment (ytsarev) [reply README.md:29]: @kuritka not sure here, used as adjective  https://en.wiktionary.org/wiki/timeproof . @donovanmuller please comment as a native speaker, thanks

**Comment by ytsarev** [comment]
@kuritka cool stuff, thanks a ton for review

**Review by k0da — APPROVED** [review]
LGTM

**Review by donovanmuller — APPROVED** [review]

**Review by kuritka — APPROVED** [review]

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (tammersaleh) [reply README.md:29]: Hey @ytsarev - is there anything we can link to that describes the DNS timeproof protocol?  Google is failing me.

**Pull request merged**